### DISCUSSION TITLE
关于 `双击全屏` 功能

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#2131] [综合讨论 / General]
今天偶然发现网页端可以双击全屏了，去反馈论坛搜了搜才发现原来我已经火星了：https://t.bilibili.com/552506920834868118

不过开了 Bilibili Evolved 后无论有无启用 `双击全屏` 功能似乎都无法使用 B 站的双击全屏功能。所以：
1. 要么删掉 `双击全屏` 功能，去兼容 B 站官方的 `双击全屏` 功能；
2. 要么维持现状，毕竟我看反馈论坛里也有好多人骂，要这功能的开关。

**Comment by the1812** [comment]
> 不过开了 Bilibili Evolved 后无论有无启用 双击全屏 功能似乎都无法使用 B 站的双击全屏功能。

听这描述像是在那个被我屏蔽的弹幕点赞层上做的(

是的话 1 好像不能做

**Reply by Geeyun-JY3** [reply]
你可以看看，如果是的话那就啥也不用干了，还实现了让用户关掉双击全屏的需求，需要也可以再开（bushi

**Reply by the1812** [reply]
确实是了, 难怪我完全不知道这回事, 自带屏蔽了(
